### ISSUE TITLE
New 0.2 Achievements print in default chat

### ISSUE THREAD

**Issue by misterff1** [alecxvs/UTChat#29]
In the 0.2 beta of JC-MP the devs changed the way some of the chat stuff works. As a result, the broadcasts for new unlocked achievements will pop up in the default chat instead of in UTChat. I don't think this is a really difficult thing to solve, but it is really annoying and would pose a serious overlay issue once 0.2 goes live,

**Comment by alecxvs** [comment]
Sounds like a bug on their part of they aren't using the prescribed interfaces for their own messages, but they haven't printed messages to the chat before anyways. I'll see if I can lobby when I get home for them to change how they print it, or provide an achievement event and let a plugin do the work instead. Or you can try and persuade them, since I haven't been in the irc in forever.

**Comment by misterff1** [comment]
Yeah I think I already found a way around it. There is a new event serverside called PlayerAchievementUnlock and altering that one changes where the message goes to. I just return false now because I don't want those messages in chat anyway, but that event provides an opportunity for a quick fix for this issue. In case you dont have much time for it (or just dont feel like it ofc :p), I can make a workaround for it and make a pull request

**Comment by alecxvs** [comment]
I have other priorities at the moment, pull requests are always welcome. I can refactor it next time I pick up the project if I need to.

**Comment by misterff1** [comment]
Alright, I will look into it then and hopefully provide a fix for it today
